packages/terra-avatar/src/Avatar.scss

```diff
@@ -0,0 +1,56 @@
+@import '~terra-mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-color: var(--terra-avatar-background-color, rgb(222, 223, 224));
+    border: var(--terra-avatar-border, 0 none);
+    border-radius: 50%;
+    box-shadow: var(--terra-avatar-box-shadow);
+    color: var(--terra-avatar-color, rgb(155, 159, 161));
+    display: inline-flex;
+    height: var(--terra-avatar-size, 3.429rem);
+    justify-content: center;
+    line-height: 0;
+    overflow: hidden;
+    width: var(--terra-avatar-size, 3.429rem);
+  }
+  
+  .avatar-icon {
+    height: 100%;
+    width: 100%;
+    
+    /* Ignoring so that single quotes can be used in svg string */
+    /* stylelint-disable string-quotes */
+    > .facility {
+      background: var(--terra-avatar-icon-facility, inline-svg("<svg data-name='Layer_1' xmlns='http://www.w3.org/2000/svg' fill='rgb(155, 159, 161)' viewBox='0 0 48 48'><path d='M33,20.2V12H15v8.2H9.8V48h12v-9h4.5v9h12V20.2H33z M17.2,42.8h-3V39h3V42.8z M17.2,33.8h-3V30h3V33.8z M25.5,33.8h-3V30h3 V33.8z M29.2,22.5h-3.8v3.8h-3v-3.8h-3.8v-3h3.8v-3.8h3v3.8h3.8V22.5z M33.8,42.8h-3V39h3V42.8z M33.8,33.8h-3V30h3V33.8z'/></svg>"));
+    }
+    
+    > .user {
+      background: var(--terra-avatar-icon-user, inline-svg("<svg data-name='Layer_1' xmlns='http://www.w3.org/2000/svg' fill='rgb(155, 159, 161)' viewBox='0 0 48 48'><path d='M24,8.6c5.1,0,9.3,4.2,9.3,9.3s-4.2,9.3-9.3,9.3s-9.3-4.2-9.3-9.3C14.8,12.8,18.9,8.6,24,8.6z M40,48v-4.7 c0-7.6-6.1-13.8-13.7-13.8c0,0,0,0,0,0h-4.5C14.1,29.6,8,35.7,8,43.3c0,0,0,0,0,0V48H40z'/></svg>"));
+    }
+    /* stylelint-enable string-quotes */
+    
+    > svg {
+      background-repeat: no-repeat;
+      background-size: contain;
+      height: 100%;
+      width: 100%;
+    }
+  }
+  
+  .avatar-text {
```

Comment:
If there are no styles applied to this class you might consider removing it and keeping `large-text` `small-text`

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).